app/controllers/hello_world_controller.rb

```diff
@@ -1,18 +1,17 @@
-# Set up the user properties. This user should appear on your LaunchDarkly users
+# Set up the context properties. This context should appear on your LaunchDarkly contexts
 # dashboard soon after you run the demo. Normal and bootstrapped flag values for
-# this user should appear in your app.
+# this context should appear in your app.
 
 class HelloWorldController < ApplicationController
   def index
-      user = {
-        key: "example-user-key",
-        name: "Sandy",
-        custom: {
-          groups: "beta_testers"
-        }
-      }
-      @user = user.to_json.html_safe
-      @client_side_id = Rails.configuration.client_side_id
-      @all_flags_state = (Rails.configuration.ld_client.all_flags_state(user, {client_side_only: true})).as_json.to_json.html_safe
+    context = LaunchDarkly::LDContext.create({
+      key: "example-user-key",
+      kind: "user",
+      name: "Sandy",
+      groups: ["beta_testers"]
+    })
+    @context = context.as_json.to_json.html_safe
```

Comment:
I know this part isn't new but I'm confused as to why _both_ `.as_json` and `.to_json` are necessary.